ProjectSidewalk/SidewalkWebpage
# 1716 add user stats table

Resolves #1716 
Resolves #1725 

Adds a `user_stat` table that has the columns `user_id`, `meters_audited`, `labels_per_meter`, `high_quality`, and `high_quality_manual`. This table is now updating nightly, and the street priority and clustering code each use the values in this table instead of calculating them again themselves.

One benefit/feature of this table is that you can set the `high_quality_manual` column to true/false if we manually look at a user's labels and decide they are high/low quality. This will then override the normal estimation of their quality. The `high_quality_manual` column is null by default.

@andrew4699 would you be able to test and review the code for this PR as well?

Testing:
1. Check out the develop branch and log in as an admin. Run single and multi-user clustering (at the /clustering endpoint). When that is done running, record how many entries are the in the `user_attribute` and `global_attribute` tables.
1. Run the street edge priority calculation (at the /audit/recalculateStreetPriority endpoint). Then record the result of the following query: `SELECT avg(priority) FROM street_edge_priority;`
1. Check out the 1716 branch and do the first two steps again. Compare the results. They should be very similar though not exact, because I switched the audited distance calculation from using completed audit tasks to missions.
1. Check that the automated nightly calculations work. In the app/utils/actor/UserStatActor.scala file, change the hours and minutes on lines 28-29 to be 2 minutes later than the current time (will need to add 7 hours to it to go from PT to UTC). Wait for that time and you should see something print in the console indicating that the calculations are being run.
1. When the calculations above are run, go back and do steps 1 and 2 again. They should have the same results as the 2nd time you ran them.
1. Test out the functionality of the `high_quality_manual` column. Take a user that has done some auditing but is marked as `high_quality = FALSE` in the database and mark the manual columns as true. Mess with the time to rerun the nightly computation again. Make sure that their `high_quality` column is now marked as true.
1. Do the opposite of the previous step for someone who was marked as high quality.
